hack/boilerplate/test/fail.go

```diff
@@ -1,8 +1,6 @@
 /*
 Copyright 2014 The Kubernetes Authors.
 
-fail
```

Comment:
I think this is required to get the boilerplate tests to fail

Reply:
Ah, that explains it. I could tell why that was there and assumed it was a copy/paste error.

I will wait to get Vince's feedback on other changes, then get this updated.